client/delegate-auth/delegate-auth.js

```diff
@@ -0,0 +1,113 @@
+// In other words, we need to have an example application which does the following:
```

Comment:
Is this file still needed? If not, could you please remove from the changes?

Reply:
it is not needed anymore indeed